__tests__/setup/raf.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+
+// http://paulirish.com/2011/requestanimationframe-for-smart-animating/
+// http://my.opera.com/emoller/blog/2011/12/20/requestanimationframe-for-smart-er-animating
+
+// requestAnimationFrame polyfill by Erik Möller. fixes from Paul Irish and Tino Zijdel
+
+// MIT license
+
+(function() {
```

Comment:
I guess we can require('raf/polyfill') instead of duplicating the polyfill; or am I wrong?